code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,153 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	w_type = RECYK_WOOD
+	frame_material = /obj/item/stack/sheet/wood
+	sheets_refunded = 2
+	autoignition_temperature = AUTOIGNITION_WOOD
+	var/obj/item/held_item
+	var/list/params_list
+	var/icon/clicked //Because BYOND can't give us runtime icon access, this is basically just a click catcher
+
+/obj/item/mounted/frame/trophy_mount/New()
+	..()
+	update_icon()
+
+/obj/item/mounted/frame/trophy_mount/Destroy()
+	if(held_item)
+		qdel(held_item)
+		held_item = null
+	..()
+
+/obj/item/mounted/frame/trophy_mount/update_icon()
+	if(!held_item)
+		overlays.len = 0
+		name = initial(name)
+		desc = initial(desc)
+	else
+		name = "[held_item.name]"
+		desc = "\A [held_item] mounted on a wooden trophy mount for display."
+		var/mutable_appearance/temp = new(held_item.appearance)
+		temp.transform = matrix()
+		temp.dir = SOUTH
+		temp.plane = FLOAT_PLANE
+		if(params_list && params_list.len)
+			var/clamp_x = clicked.Width() / 2
+			var/clamp_y = clicked.Height() / 2
+			temp.pixel_x = Clamp(text2num(params_list["icon-x"]) - clamp_x, -clamp_x, clamp_x)
+			temp.pixel_y = Clamp(text2num(params_list["icon-y"]) - clamp_y, -clamp_y, clamp_y)
+
+		overlays += temp
+
+	clicked = new/icon(src.icon, src.icon_state, src.dir)
+
+/obj/item/mounted/frame/trophy_mount/examine(mob/user)
+	..()
+	if(held_item)
+		held_item.examine(user, show_name = FALSE)
+
+/obj/item/mounted/frame/trophy_mount/attackby(obj/item/weapon/W, mob/user, params)
+	..()
+	params_list = params2list(params)
+	if(iswrench(W))
```

Comment:
This should be handled instead by making the `..()` return `TRUE` if it's handled and checking that instead.